### DISCUSSION TITLE
Hitbox for interacting with characters too big

### DISCUSSION THREAD

**Discussion by bookslayer10** [FakeFishGames/Barotrauma#10135] [Ideas and suggestions]
I often find that when I hit the button for the health menu, I open up the menu for characters on the ground below me instead. I've turned off aim assist for clicking on items completely, but the hitbox for pressing the heal hotkey to heal another creature is still massive; I'm able to open up the healing prompt for a collapsed character by clicking at head height.

This has actually lead to me applying first aid to an enemy rather than myself because I select them by mistake, and while I blame this on my inexperience with the system, it is still a fair issue.

**Comment by MaaiG** [comment]
**1. I also believe that the interaction zone should be reduced.**

**2. The interaction distance should be reduced!**
For example: the character should only press the button when he came close to it.

**3. Separate interaction with cabinets, devices, characters, etc.**
Many submarines have a dense arrangement of internal devices and other objects. This leads to frequent erroneous interactions with closely spaced or overlapping objects. Some object categories can be interactive only when holding an modifier key (Shift, Ctrl, Alt). So you will never choose a periscope if you try to open the cabinet behind it.
